skrf/media/cpw.py

```diff
@@ -25,50 +24,198 @@
 
 class CPW(Media):
     """
-    A Coplanar Waveguide.
-
-    This class was made from the technical documentation [#]_ provided
-    by the qucs project [#]_ .
-    The variables  and properties of this class are coincident with
-    their derivations.
+    A coplanar waveguide transmission line is defined in terms of width,
+    spacing, and thickness on a given relative permittivity substrate of a
+    certain height. The line has a conductor resistivity and a tangential loss
+    factor. The backside of the strip can be made of air or metal (grounded 
+    coplanar waveguide).
+
+    This class is highly inspired by the technical documentation [QUCSa]_
+    and sources provided by the qucs project [QUCSb]_ .
+    
+    In addition, Djordjevic [DBLS01]_ /Svensson [SvDe01]_  wideband debye dielectric
+    model is considered to provide a more realistic modelling of broadband
+    microstrip with causal time domain response.
+    
+    A compatibility mode is provided to mimic the behaviour of QUCS or of
+    Keysight ADS. There is known differences in the output of these
+    simulators.
+    
+    The quasi-static models of characteristic impedance and effective
+    permittivity give the value at zero frequency. The dispersion models
+    compute frequency-dependant values of these variables.
+    
+    * Quasi-static characteristic impedance and effective permittivity model
+      use [GhNa84]_ and [GhNa83]_. The models are corrected to account for
+      strip thickness using a first-order approach described in [GGBB96].
+      A comparison shows that ADS simulator uses another thickness correction
+      method that is according to ADS doc based on [Cohn60]. This second method
+      is not implemented in skrf.
+    
+    * Frequency dispersion of impedance and effective permittivity model use
+      [FGVM91]_ and [GMDK97]_.
+    
+    * Loss model is computed using Wheeler's incremental inductance rule
+      [Whee42]_ applied to coplanar waveguide by [OwWu58]_ and [Ghio93]_.
 
     Parameters
     ----------
     frequency : :class:`~skrf.frequency.Frequency` object, optional
-        frequency band of the media. The default is None.
-    z0 : number, array-like, optional
-        the port impedance for media. The default is None.
-        Only needed if  its different from the characteristic impedance
-        of the transmission.
-    w : number, or array-like, optional
-        width of center conductor, in m. Default is 70.
+        frequency band of the media
```

Comment:
```suggestion
        frequency band of the media. The default is None.
```

Reply:
Done and added even more informations about defaults.